EntryControl/EntryControl/SimulateMovement.cs

```diff
@@ -0,0 +1,100 @@
+﻿using EntryControl.Data;
+using EntryControl.Models;
+
+namespace EntryControl
+{
+    public class SimulateMovement
+    {
+        private  DateTime _timeFrom = new(2022, 1, 1);
+        private DateTime _timeTo = new(2022, 2, 28);
+        private Random _rnd = new();
+        public SimulateMovement() 
+        {
+        }
+        public SimulateMovement(DateTime from, DateTime to)
```

Comment:
Nematau jokios priezasties from ir to inicializuoti per konstruktoriu ir padaryti available visai klasei, kai jie naudojami tik viename metode.  As juos paduociau kaip metodo Simulate parametrus.